src/shared/email/email-template-loader.ts

```
@@ -0,0 +1,53 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+export class EmailTemplateLoader {
+  private static templatesPath = path.join(__dirname, 'templates');
+
+  static loadTemplate(templateName: string): string {
+    const templatePath = path.join(this.templatesPath, `${templateName}.html`);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)